bluesky_kafka/tests/test_publisher.py

```diff
@@ -11,15 +11,28 @@ def test_producer_config():
         topic=topic,
         bootstrap_servers="1.2.3.4:9092",
         key="test.producer.config",
-        # work with a single broker
         producer_config={
             "bootstrap.servers": "5.6.7.8:9092",
-            "acks": 1,
-            "enable.idempotence": False,
-            "request.timeout.ms": 5000,
         },
     )
 
     assert (
         publisher._producer_config["bootstrap.servers"] == "1.2.3.4:9092,5.6.7.8:9092"
     )
+
+
+def test_redact_password_from_str_output():
+    topic = "test.redact.password"
+    publisher = Publisher(
+        topic=topic,
+        bootstrap_servers="1.2.3.4:9092",
+        key="test.redact.password",
```

Comment:
Should it be the same key as in the Consumer test above (for consistency)?
```suggestion
        key="test-redact-password",
```

Reply:
The producer "key" and consumer "group_id" are not related. Here is my proposal for making them more consistent anyway:
```
    key="test-redact-password-key"
    group_id="test-redact-password-group"
```